### PR TITLE
KM review (part 2) -- Editorial pass on Section 4.4

### PR DESCRIPTION
**Pull request by thomas-fossati**
https://mailarchive.ietf.org/arch/msg/rats/7-UJAqHPMLhrGCk9tyh3JqGu8rw/

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -467,7 +467,8 @@ Device, each slot is an Attester, and the main slot is the lead Attester.
 Another example is a multi-chassis router composed of multiple single carrier-grade routers.
 The multi-chassis router provides higher throughput by interconnecting
 multiple routers and can be logically treated as one router for simpler management.
-Among these routers, there is only one main router that connects to the Verifier.
+A multi-chassis router provides a management point and is the only one that
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:470]: My reading of the old text is that the new text is incorrect.  The multi-chassis router internally includes multiple routers and one of the internal one is referenced, not the overall multi-chassis router.  Is my reading correct?

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -467,7 +467,8 @@ Device, each slot is an Attester, and the main slot is the lead Attester.
...
+connects to the Verifier.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:471]:
```suggestion
A multi-chassis router provides a management point and is the one that connects to the Verifier.
```

**Comment by mcr** [comment]
@Eliot Lear.

File: draft-ietf-rats-architecture.md (hunk 1 above)
Comment (elear) [reply draft-ietf-rats-architecture.md:470]: I will check within Cisco on this.

Comment (William-PanWei) [reply draft-ietf-rats-architecture.md:470]: The multi-router is a logical concept, it internally includes multiple routers, and these routers are not just the redundancy relationship. From the perspective of the physical routers, there is one router connecting to the Verifier. From the perspective of the logical multi-router, you may say that there is a management point connecting to the Verifier, and actually this point is one router.

**Pull request merged**